packages/expo-media-library/android/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,5 @@
 <manifest package="expo.modules.medialibrary" xmlns:android="http://schemas.android.com/apk/res/android">
   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+  <uses-permission android:name="android.permission.ACCESS_MEDIA_LOCATION" />
```

Comment:
Maybe it's better to use a config plugin for this so it can be more visible, and easier to remove. Otherwise users will be getting a new permission for a library they've already implemented.

Reply:
okay will do - I would like to understand better though -- doesn't a config plugin accomplish the same thing? e.g it will add this new permission regardless?